src/uct/ib/base/ib_iface.c

```diff
@@ -392,6 +392,22 @@ static ucs_status_t uct_ib_iface_init_lmc(uct_ib_iface_t *iface,
     return UCS_OK;
 }
 
+static int uct_ib_max_cqe_size()
+{
+    static int max_cqe_size = -1;
+
+    if (max_cqe_size == -1) {
+        max_cqe_size =
+#ifdef __aarch64__
```

Comment:
Is this really needed ? You detect the PCIe vendor and root complex. I'm not sure if this has to be arch specific

Reply:
yes, i guess we can remove it